packages-ts/gauntlet-terra/src/commands/logger.ts

```diff
@@ -1,5 +1,5 @@
 import { logger } from '@chainlink/gauntlet-core/dist/utils'
-import { AddressBook } from './addressBook'
+import { AddressBook } from '@chainlink/gauntlet-core'
 import { assertions } from '@chainlink/gauntlet-core/dist/utils'
 
 type COLOR = 'red' | 'green' | 'blue' | 'yellow' | 'cyan' | 'magenta'
```

Comment:
The `TerraLogger` can be replaced by the new `Logger` from core right?

Reply:
yeah there's another PR open to address that - I still need to update that.

Reply:
Let's include both changes in this PR

Reply:
cool I just rebased it with the new changes! So this should be good now @RodrigoAD